pypeit/core/framematch.py

```diff
@@ -25,6 +25,9 @@ def __init__(self):
         # order as part of its data model. When/if we require python
         # 3.7, we can remove this (and other) OrderedDict usage in favor
         # of just a normal dict.
+
+        # TODO JFH: I don't think we ever use pinhole. Should we remove it.
```

Comment:
Question for @rcooke-ast

Reply:
Pinhole is very useful for echelle when orders are close to overlapping. I've come across cases (e.g. folks that observe stars) where these frames can be helpful. Does anyone know when the D5 decker last used on HIRES? I can't seem to do this search on KOA... So, pinholes are helpful for HIRES. Are pinhole's used for APF @bpholden?

Reply:
We can leave them in, but again we don't use them anywhere yet.